Onboarding/Sources/Onboarding/Views/OnboardingContainerView.swift

```diff
@@ -0,0 +1,61 @@
+import SwiftUI
+
+struct OnboardingContainerView: View {
+    @StateObject private var viewModel: OnboardingViewModel
+    private var themeColor = UIColor(Color("AccentColor", bundle: Bundle.module))
+    private var textColor =  UIColor(Color("PrimaryColor", bundle: Bundle.module))
+    @Environment(\.verticalSizeClass) var verticalSizeClass
+    init(viewModel: OnboardingViewModel) {
+        _viewModel = StateObject(wrappedValue: viewModel)
+        UIPageControl.appearance().currentPageIndicatorTintColor = themeColor
```

Comment:
ooof, is this going to affect the entire app?

Reply:
This has now been moved to the uikit page controller wrapper class